src/pages/Wallet/WalletHomePage.tsx

```diff
@@ -206,6 +190,41 @@ const WalletHomePage = () => {
   )
 }
 
+const IOList = ({
+  currentAddress,
+  isOut,
+  transaction
+}: {
+  currentAddress: string
+  isOut: boolean
+  transaction: Transaction
+}) => {
+  const io = (isOut ? transaction.outputs : transaction.inputs) as Array<Output | Input>
+  const genesisTimestamp = 1231006505000
+
+  if (io && io.length > 0) {
```

Comment:
Based on line `202`, the `io` variable is either an `Array` of `Output`s or an `Array` of `Input`s, or an empty `Array`. None of this values is a falsy value so I think we don't need to check `if (io && `. Is there a case where `transaction.outputs` AND `transaction.inputs` are [both undefined](https://github.com/alephium/alephium-js/blob/master/api/api-explorer.ts#L83-L84)?

Reply:
My casting is a bit "optimistic": if we look at our API typings, I should cast `as Array<Output | Input> | undefined` and keep the `io &&` part. @tdroxler Do you know a case where bot outputs and inputs would be undefined?

Reply:
mmmm no I don't see it as a valid case BUT it could be that the explorer-backend wrongly sync and suddenly return no outputs nor inputs. So maybe it's safer to handle that case?
Now that both of you are type lovers: that's why some libraries are proposing some `NonEmptyArray` type, where you are 100% sure you get at least one element, but we don't have this for now.

Reply:
Thanks @tdroxler! As we don't have this for now, I'm for keeping it safe and continue checking for `io` definition.